app/upgrades/signetlaunch/upgrades.go

```diff
@@ -79,6 +101,115 @@ func LoadBTCHeadersFromData() ([]*btclighttypes.BTCHeaderInfo, error) {
 	return gs.BtcHeaders, nil
 }
 
+// LoadSignedFPsFromData returns the finality providers from the json string.
+// It also verifies if the msg is correctly signed and is valid to be inserted.
+func LoadSignedFPsFromData(cdc codec.Codec, txJSONDecoder sdk.TxDecoder) ([]*btcstktypes.MsgCreateFinalityProvider, error) {
+	buff := bytes.NewBufferString(SignedFPsStr)
```

Comment:
Could the json string load from a file?

Reply:
Nop :/ otherwise, it doesn't compile together with the binary, so you would need to also send this json file to who wants to upgrade and put it aside the binary when the upgrade is happening